app/scripts/migrations/125.ts

```diff
@@ -0,0 +1,50 @@
+import { cloneDeep } from 'lodash';
+import { hasProperty, isObject } from '@metamask/utils';
+
+type VersionedData = {
+  meta: { version: number };
+  data: Record<string, unknown>;
+};
+
+export const version = 125;
```

Comment:
Since we're planning on including this in a hotfix, we should make this v120.2 instead. v12 is on v120.1. Otherwise we'll need to re-order the migration when cherry-picking it, then re-order it back again when merging back into develop. This process is time consuming and has led to bugs in the past.

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/26265/commits/28414ebceccbabf3bea0032ef095ac5b86d9cb52